docs/reference/query-dsl/minimum-should-match.asciidoc

```diff
@@ -1,13 +1,14 @@
 [[query-dsl-minimum-should-match]]
 == Minimum Should Match
 
+The property `minimum_should_match` can be used with a bool `should` that has multiple clauses. Since multiword `match` and `multi-match` queries simply wrap the generated term queries in a bool query, they can use `minimum_should_match` as well.
 The `minimum_should_match` parameter possible values:
 
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Type |Example |Description
 |Integer |`3` |Indicates a fixed value regardless of the number of
-optional clauses.
+optional clauses. Zero is allowed.
```

Comment:
While zero is allowed, mentioning it this prominently here might raise the question of what is does, which is a much more complicated story that might be better answered elsewhere and in more depth (e.g. when there are no must or filter clauses, `minimum_should_match` of 0 doesn't return all documents as one might expect).
Also this contradicts  `a value greater than the number of optional clauses, or a value less than 1 will never be
used. ` in the last paragraph on this page.